types/murmurhash3js-revisited/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for murmurhash3js-revisited 3.0
+// Project: https://github.com/cimi/murmurhash3js-revisited#readme
+// Definitions by: Carlo Francisco <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface X86 {
+    hash32: (bytes: Uint8Array) => number;
+    hash128: (bytes: Uint8Array) => string;
+}
+interface X64 {
+    hash128: (bytes: Uint8Array) => string;
+}
+export const x86: X86;
+export const x64: X64;
+
+declare const MurmurHash3: {
+    x86: X86;
+    x64: X64;
+};
+export default MurmurHash3;
```

Comment:
I don't think this is correct, there is an explcit comment about UMD exort:
https://github.com/cimi/murmurhash3js-revisited/blob/de8d6862b5b6507a96c0114547be7944dedd8d86/lib/murmurHash3js.js#L549-L550

Reply:
Good catch, I'll try to fix it this weekend

Reply:
@peterblazejewicz Fixed with 30aba4945.

It's strange, because the [documentation](https://cimi.io/murmurhash3js-revisited/) suggests there's a default export. But I don't see it anywhere in the code. Unless I missed something...

![image](https://user-images.githubusercontent.com/217960/117575131-7e9dc900-b0ae-11eb-8673-97e758f49cc1.png)

Reply:
The docs are probably done based on experience of someone using Babel with defaults, works a treat.
TS provides support for this syntax using `esModuleInterop`. But we are trying to promote a 'works everywhere' syntax here. See details (inluding `works everyewhere` paragraph) here:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html#default-exports